.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
```

Comment:
interesting why these files are not ignored by git.

Reply:
Yes, sure. I'll see the course record to see where the mistake occurred.